static/app/components/charts/chartWidgetLoader.tsx

```diff
@@ -228,8 +229,13 @@ export function ChartWidgetLoader(props: Props) {
     // eslint-disable-next-line no-console
     console.error(error);
     captureException(error);
-    return <Placeholder height="100%" error={t('Error loading widget')} />;
+    return <ErrorPlaceholder height="100%">{t('Error loading widget')}</ErrorPlaceholder>;
   }
 
   return <Component {...props} chartRef={props.ref} />;
 }
+
+const ErrorPlaceholder = styled(Placeholder)`
```

Comment:
This is not what we should be using the component for. Given that there were only two instances where this was setup, I've went ahead and made these custom styled. At some point in the future, we will review all usages of styled(CoreComponent), and look at this in more detail, but for now, this is at least not something that we support ootb